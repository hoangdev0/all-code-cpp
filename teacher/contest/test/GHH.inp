5
8
1
12
80
88
