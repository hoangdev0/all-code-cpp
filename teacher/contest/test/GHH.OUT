3
12
80
88
